package.json

```diff
@@ -17,7 +17,7 @@
     "test:ssr": "jest --maxWorkers 2 --config ./jest.config.node.ts",
     "test:watch": "jest --watch",
     "test:coverage": "jest --coverage",
-    "lint": "eslint {src,tests,stories}/**/*.{ts,tsx}",
+    "lint": "eslint \"{src,tests,stories}/**/*.{ts,tsx}\"",
```

Comment:
@streamich you might want to use `\"` here in your version of the fix. This prevents the following error:

```
yarn lint                                                                                                                                                ✘ INT 22:12:42
yarn run v1.22.11
$ eslint {src,tests,stories}/**/*.{ts,tsx}

Oops! Something went wrong! :(

ESLint: 7.26.0

No files matching the pattern "stories/**/*.ts" were found.
Please check for typing mistakes in the pattern.

error Command failed with exit code 2.
```

Looks like without extra `"`, the glob is applied in shell, thus giving `eslint` a bunch of arguments with all possible combinations. There’s nothing under the `"stories/**/*.ts"` pattern, so ESLint exits with error.